metadata-ingestion/.gitignore

```diff
@@ -4,6 +4,7 @@ output
 src/datahub/metadata/
 pvenv36/
 bq_credentials.json
+/tmp
```

Comment:
did you mean tmp/ ?

Reply:
The leading slash refers to the current directory in gitignore syntax, so this should be fine